src/fileseq/frameset.py

```diff
@@ -846,7 +851,7 @@ def isFrameRange(frange):
         """
         # we're willing to trim padding characters from consideration
         # this translation is orders of magnitude faster than prior method
-        frange = str(frange).translate(None, ''.join(PAD_MAP.keys()))
+        frange = ''.join([ch for ch in asString(frange) if not ch in PAD_MAP])
```

Comment:
Same comment as previous. Can we branch 2/3 logic and use:

```
asString(frange).translate(str.maketrans('', '', ''.join(PAD_MAP)))
```

Reply:
👍